elastic-otel/docker-compose.yml

```
@@ -0,0 +1,76 @@
+services:
+  example-api:
+    build:
+      context: ./src/example-api
+      dockerfile: ./Dockerfile
+    container_name: example-api
+    environment:
+      - ASPNETCORE_ENVIRONMENT=Production
+      - OTEL_EXPORTER_OTLP_ENDPOINT=http://otel-collector:4317
+      - OTEL_EXPORTER_OTLP_PROTOCOL=grpc
+    volumes: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)